examples/bri-3/src/bri/transactions/agents/transactionStorage.agent.ts

```diff
@@ -1,30 +1,43 @@
 import { Injectable, NotFoundException } from '@nestjs/common';
+import Mapper from '../../utils/mapper';
 import { PrismaService } from '../../../../prisma/prisma.service';
 import { NOT_FOUND_ERR_MESSAGE } from '..//api/err.messages';
 import { Transaction } from '../models/transaction';
+import { getType } from 'tst-reflect';
 
 @Injectable()
 export class TransactionStorageAgent extends PrismaService {
+  constructor(private readonly mapper: Mapper) {
+    super();
+  }
+
+  async getAllTransactions(): Promise<Transaction[]> {
```

Comment:
Just a note, we deliberately left our the get all endpoint because we are unsure of the scope for this query. I am fine with including it here.

Reply:
> @Manik-Jain what about automapper, is there a decision? imo we should decide on that before adding additional code to current mapper

Yes, we have agreed to change response interfaces to classes and then we can have auto-mapper changes. 
https://github.com/eea-oasis/baseline/issues/558